spec/features/studios/index_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe "the studios index" do
+  before(:each) do
+    @uni = Studio.create!(name: "Universal Studios", location: "Hollywood")
+    @dis = Studio.create!(name: "Disney", location: "LA")
+    
+    @good = @uni.movies.create!(title: "Goodwill Hunting", creation_year: "1992", genre: "drama")
+    @mib = @uni.movies.create!(title: "Men In Black", creation_year: "1997", genre: "animation")
+    @toy = @dis.movies.create!(title: "Toy Story", creation_year: "1995", genre: "animation")
+
+    @tom = Actor.create!(name: "Tom Hanks", age: 65)
+    @matt = Actor.create!(name: "Matt Damon", age: 58)
+    @will = Actor.create!(name: "Will Smith", age: 54)
+
+    @good_matt = MoviesActor.create!(movie: @good, actor: @matt)
+    @toy_tom = MoviesActor.create!(movie: @toy, actor: @tom)
+    @mib_will = MoviesActor.create!(movie: @mib, actor: @will)
+
+    visit "/studios"
+  end
+
+  describe "the studios index page" do 
+    it "lists all the studios with their attributes" do
```

Comment:
We could use some `within` blocks to organize this a little better and ensure things are appearing where we want them to appear.